NsqSharp/Consumer.cs

```diff
@@ -780,6 +788,7 @@ private void connectToNsqd(string addr)
             }
 
             // pre-emptive signal to existing connections to lower their RDY count
+            _perConnMaxInFlight = 0;
```

Comment:
Agreed. I renamed it `_perConnMaxInFlightOverride`. I'm open to suggestions on naming and the general approach.